spec/unit/store/Preferences/Appearance.spec.js

```diff
@@ -0,0 +1,132 @@
+        timeFormat: TimeFormat.Absolute.value,
+        customThemeColor: LightTheme,
+        font: DefaultFonts[0]
+      },
+      fonts: []
+    }
+  })
+  describe('mutations', () => {
+    describe('updateAppearance', () => {
+      it('should be changed', () => {
+        Appearance.mutations.updateAppearance(state, {
+          theme: Theme.Dark.key
+        })
+        expect(state.appearance.theme).toEqual(Theme.Dark.key)
+      })
+    })
+    describe('updateFonts', () => {
+      it('should be changed', () => {
+        Appearance.mutations.updateFonts(state, ['font'])
+        expect(state.fonts).toEqual(['font'])
+      })
+    })
+  })
+
+  describe('actions', () => {
+    describe('loadAppearance', () => {
+      it('error', async () => {
+        ipcMain.once('get-preferences', (event, _) => {
+          event.sender.send('error-get-preferences', new LoadAppearanceError())
+        })
+        const commitMock = jest.fn()
+        await Appearance.actions.loadAppearance({ commit: commitMock })
+          .catch((err) => {
+            expect(err instanceof LoadAppearanceError).toEqual(true)
+          })
+      })
+      it('success', async () => {
+        const conf = {
+          appearance: {
+            theme: Theme.Dark.key
+          }
+        }
+        ipcMain.once('get-preferences', (event, _) => {
+          event.sender.send('response-get-preferences', conf)
+        })
+        const commitMock = jest.fn()
+        const c = await Appearance.actions.loadAppearance({ commit: commitMock })
+        expect(c).toEqual(conf)
+        expect(commitMock).toHaveBeenCalledWith('updateAppearance', conf.appearance)
+      })
+    })
+    describe('loadFonts', () => {
+      it('error', async () => {
+        ipcMain.once('list-fonts', (event, _) => {
+          event.sender.send('error-list-fonts', new ListFontsError())
+        })
+        const commitMock = jest.fn()
+        await Appearance.actions.loadFonts({ commit: commitMock })
+          .catch((err) => {
+            expect(err instanceof ListFontsError).toEqual(true)
+          })
+      })
+      it('success', async () => {
+        ipcMain.once('list-fonts', (event, _) => {
+          event.sender.send('response-list-fonts', ['fonts'])
+        })
+        const commitMock = jest.fn()
+        const fonts = await Appearance.actions.loadFonts({ commit: commitMock })
+        expect(fonts).toEqual(['fonts'])
+        expect(commitMock).toHaveBeenCalledWith('updateFonts', [DefaultFonts[0]].concat(['fonts']))
+      })
+    })
+  })
+  describe('updateTheme', () => {
+    it('error', async () => {
+      ipcMain.once('update-preferences', (event, _) => {
+        event.sender.send('error-update-preferences', new UpdateThemeError())
+      })
+      const dispatchMock = jest.fn()
+      const commitMock = jest.fn()
+      await Appearance.actions.updateTheme({ dispatch: dispatchMock, commit: commitMock, state: state}, Theme.Dark.key)
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - A space is required before '}'. (object-curly-spacing)